.gitignore

```
@@ -32,4 +32,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 
-.idea
+.idea/
```

This chamge is not required, you can leave it as is.
```suggestion
.idea
```